### ISSUE TITLE
When Richedit Class is being used by some Windows aPP and user press ESC, NVDA is causing Access violation inside Richedit library or directly inside .exe application processes.

### ISSUE THREAD

**Issue by JanuszChmiel** [nvaccess/nvda#6604]
Dear developers,
I AM starting with new group of issues, which have been very probably caused by some internal changes of NVDA, very probably after NVDA 2016.2 have been released.
This issue is very un soft to users. When I AM using latest version of Skype while using Windows 10 home edition X64 bit edition. NVDA is causing Access violation address fault inside some of Richedit library. When? When focus is located on The edit box of Skype chat Window. When? When I press ESC key. Error is occurring randomly, not every time. This is general NVDA bug, which is also presented while using Miranda NG stable. So if focus changes from Richedit to listbox with contacts , NVDA is causing severe crash inside running App. No matter, if App have been compiled by using some C++ or C language compiler, or if app have been build and compiled by Borland Delphi or by other Pascal compiler. This bug is presented randomly in Skype. I have accessibility support in Skype enabled. And I do not use special addon for Skype for NVDA. Such as Skype addon. I will try to help you debugging. Try to press enter on some Skype contacts and try to press Esc when focus is inside Richedit. Must I run NVDA as administrator? So do I have to modify NVDA desktop link so it will be run always as administrator? I Am using Windows defender as my anti spy ware and anti virus solution. I Am using OEM preinstalled version of Windows 10 included with Lenovo laptop. I can try to remove Uac .sys drivers instructions for load from registry, if it will not help. But I do not think it so. May be, that C++ application named Wonderful icon can cause this issue. I will intensively test it. Because this app contain keyhook library, which is intensively monitoring hod keys. But build in hod keys detector inside Explorer.exe is too slow. I Am using Wonderful icon to start Core media player actions while transcribing The audio record to text.

**Comment by jcsteh** [comment]
We're pretty certain this is caused by a bug in RichEdit's UI Automation implementation. Certainly, none of NVDA's code is anywhere in the crash stack. Anyway, it's a duplicate of #6532 (which is in turn probably related to #6579). Thanks.
